data/data/install.openshift.io_installconfigs.yaml

```
@@ -1098,7 +1098,7 @@ spec:
                     description: ClusterOSImage is either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes, or an existing Glance image name.
                     type: string
                   computeFlavor:
-                    description: FlavorName is the name of the compute flavor to use for instances in this cluster.
+                    description: 'DeprecatedFlavorName is the name of the compute flavor to use for instances in this cluster. Deprecated: use FlavorName in DefaultMachinePlatform to define default compute flavor.'
```

The term `compute` can be confusing, I suggest removing it. 
```suggestion
                    description: 'DeprecatedFlavorName is the name of the flavor to use for instances in this cluster. Deprecated: use FlavorName in DefaultMachinePlatform to define default flavor.'
```

---

done